platform/vs/docker-sonic-vs/Dockerfile.j2

```diff
@@ -106,6 +106,7 @@ COPY ["files/configdb-load.sh", "/usr/bin/"]
 COPY ["files/arp_update", "/usr/bin/"]
 COPY ["files/buffers_config.j2", "files/qos_config.j2", "/usr/share/sonic/templates/"]
 COPY ["files/sonic_version.yml", "/etc/sonic/"]
+COPY ["database_config.json", "/etc/default/sonic-db/"]
 
 # Workaround the tcpdump issue
```

Comment:
Will you add
```
VOLUME /var/run/redis/sonic-db/
```

Reply:
We don't need add VOULME again, it already added when we start the dvs in conftest.py.

Reply:
Dockerfile `VOLUME` is different from `docker run -v`. If you mark it as `VOLUME`, anything already there (built into docker image) will be copied out to host when you mount. I believe this is useful in some test cases.

Reply:
We can verify and use this VOLUME MODE later. For now, there is nothing in /var/run/redis/ built into docker image.  And today we only use this /var/run/redis after docker image is running. I prefer not to change current behavior which is not related to multiDB changes.